.github/actions/paths-filter/action.yml

```diff
@@ -17,6 +17,13 @@ outputs:
         github.event_name == 'schedule' ||
         steps.filter-common.outputs.github-actions-change == 'true'
       }}
+  check-copilot-commits:
+    description: Output whether `check-copilot-commits` should be run based on GitHub events
+    value: >-
+      ${{
+        github.event_name == 'pull_request' ||
+        github.event_name == 'push'
```

Comment:
Push will only be effective on `stable/*` branches and the `main` branch, as we don't run `on: push` for any other branches in the `ci.yml`.

Does this align with your intention? I'm asking because running this on `main`, after the merge and merge queue will probably not bring much value?